# -Regulation_PTEN_KyriePappas2018
Paper by Pappas, K. et al, 2018 related about the epigenetic regulation of PTEN expression in breast cancer

We provide here different command lines to visualise the genomic and epigenomic regulation loci of PTEN using R bioconductor packages coMET and Gviz and published data :
1) COSMIC data for somatic mutations associated with Breast cancer (downloaded on the March,26th 2018 and are from the release v84 (13th February 2018), https://cancer.sanger.ac.uk/cosmic)
2) ENSEMBL for the exonic/intronic structures of different genes (Version 91 of GRCh37.13, https://grch37.ensembl.org/index.html)
3) Pappas et al., 2017, "p53 Maintains Baseline Expression of Multiple Tumor Suppressor Genes" http://mcr.aacrjournals.org/content/early/2017/05/06/1541-7786.MCR-17-0089 for binding site of P53 in MCF10A
4) chromHMM for 5 human normal breast cell-lines (MCF10A, basal, stromal, luminal mature, and lumninal progenitor) published by Pellacani et al., 2017, "Analysis of Normal Human Mammary Epigenomes Reveals Cell-Specific Active Enhancer States and Associated Transcription Factor Networks". Processed signal tracks are also available at http://www.epigenomes.ca .
5) chromHMM for HMEC cell-line from ENCODE/ROADMAP Epigenomics, March 2012 Freeze (GEO sample accession: GSM936084)
6) ATAC-seq data in MCF10A cell-line from Liu et al, 2017, "Identification of breast cancer associated variants that modulate transcription factor binding" (GEO accession number: GSE89013)
7) DNase-seq of MCF7 cell-line (GEO accession number: GSM736588) and HMEC (GEO accession number: GSM736634) from ENCODE 

 More information about the extraction data and computational analysis in the pdf associated here.
